frontend/public/components/routes.tsx

```diff
@@ -226,20 +226,21 @@ const RouteIngressStatus: React.SFC<RouteIngressStatusProps> = ({ingresses}) =>
   <React.Fragment>
     {_.map(ingresses, (ingress) =>
       <div key={ingress.routerName} className="co-m-route-ingress-status">
-        <h1 className="co-section-title">Router: {ingress.routerName}</h1>
+        <SectionHeading text={`Router: ${ingress.routerName}`} />
         <dl>
           <dt>Hostname</dt>
           <dd>{ingress.host}</dd>
           <dt>Wildcard Policy</dt>
           <dd>{ingress.wildcardPolicy}</dd>
         </dl>
-        <h2 className="co-section-title-secondary">Conditions</h2>
+        <h3 className="co-section-heading-secondary">Conditions</h3>
```

Comment:
I can see this getting confusing, though.  I'd prob rather see us always using `<SectionHeading />` but with a prop to toggle `h1/h2/h3/h4` & the appropriate default classes...

I don't think you need to do anything for this PR, just mentioning it.